.pre-commit-config.yaml

```
@@ -10,7 +10,7 @@ repos:
       - id: double-quote-string-fixer
 
   - repo: https://github.com/psf/black
-    rev: 25.1.0
+    rev: 25.1.1
```

Interestingly, 25.1.1 doesn't exist.
```suggestion
    rev: 25.1.0
```